urdf/sdf_extensions.md

```diff
@@ -37,7 +37,7 @@ results in:
 
 ```xml
 <!--SDFormat-->
-<sdf version='1.9'>
```

Comment:
I don't think this is necessary since the tutorial applies to SDFormat versions 1.9 and up. If someone was using an older version of libsdformat, it would generate 1.9.

Reply:
Oh ok, sounds good to me. Closing the PR then.